docs/getting_started.rst

```diff
@@ -0,0 +1,131 @@
+:orphan:
+
+###############
+Getting started
+###############
+
+Installation
+============
+
+Qiskit Nature depends on the main Qiskit package which has its own
+`Qiskit Getting Started <https://qiskit.org/documentation/getting_started.html>`__ detailing the
+installation options for Qiskit and its supported environments/platforms. You should refer to
+that first. Then the information here can be followed which focuses on the additional installation
+specific to Qiskit Nature .
+
+Qiskit Nature has some functions that have been made optional where the dependent code and/or
+support program(s) are not (or cannot be) installed by default. These include, for example,
+classical libary/programs for molecular problems.
```

Comment:
```suggestion
classical library/programs for molecular problems.
```

Reply:
I have the spellchecker turned on locally. I missed the above one - its right over a highlighted block on my machine so I did not notice the wiggly underlining. But as Manoel mentioned if we switch spellchecked to sphinx output it would take care of mistakes like that!